golden_response/staging/get_related_locations/county.json

```diff
@@ -1,18 +1,18 @@
 {
   "Count_Person": {
     "rankFromTop": 17,
-    "rankFromBottom": 3133
+    "rankFromBottom": 3132
   },
   "Median_Age_Person": {
-    "rankFromTop": 2583,
-    "rankFromBottom": 603
+    "rankFromTop": 2572,
+    "rankFromBottom": 613
   },
   "Median_Income_Person": {
     "rankFromTop": 28,
-    "rankFromBottom": 3158
+    "rankFromBottom": 3157
   },
   "UnemploymentRate_Person": {
-    "rankFromTop": 675,
-    "rankFromBottom": 2512
+    "rankFromTop": 1016,
```

Comment:
This seems suspicious... I'll check on this.

Reply:
This is also explained by the choice of time-series, since unemployment-rate fluctuated a lot recently.  Now we prefer the series which has the latest date while computing ranking, so I think this should be more accurate.